src/ResourceManagement/Compute/ComputeManagement/generate.cmd

```diff
@@ -4,7 +4,7 @@
 ::
 
 @echo off
-set autoRestVersion=0.11.0-Nightly20150907
+set autoRestVersion=0.11.0-Nightly20150911
```

Comment:
In case it helps, you can checkout the most recent build status from: http://azuresdkci.cloudapp.net/view/3-AutoRest/job/autorest-nightly.